.github/component_owners.yml

```diff
@@ -9,7 +9,6 @@ components:
   detectors/node/opentelemetry-resource-detector-gcp:
     - aabmass
     - punya
-    # Unmaintained?
```

Comment:
looks like this was here even though we have two owners, so I removed the comment.